src/main/java/com/github/igotyou/FactoryMod/factories/FurnCraftChestFactory.java

```diff
@@ -166,7 +167,13 @@ public void attemptToActivate(Player p, boolean onStartUp) {
 		//ensure we have fuel
 		if (!pm.powerAvailable(1)) {
 			if (p != null) {
-				p.sendMessage(ChatColor.RED + "Failed to activate factory, there is no fuel in the furnace");
+				ItemStack fuel = ((FurnacePowerManager) pm).getFuel().clone();
+				if (fuel != null) {
+					String fuelName = ItemNames.getItemName(fuel);
+					p.sendMessage(ChatColor.RED + "Failed to activate factory, there is no fuel (" + fuelName.toLowerCase() + ") in the furnace");
```

Comment:
You shouldn't have to lower case the name from ItemNames, it should always be correctly capitalized